tests/vm_test.go

```diff
@@ -1267,3 +1255,11 @@ func createRunningVM(virtClient kubecli.KubevirtClient, template *v1.VirtualMach
 	Expect(err).ToNot(HaveOccurred())
 	return vm
 }
+
+func powerOff(vmi *v1.VirtualMachineInstance) {
+	// Can't use SafeExpectBatch since we may not get a prompt after the command
+	ExpectWithOffset(1, console.ExpectBatch(vmi, []expect.Batcher{
+		&expect.BSnd{S: "sudo poweroff\n"},
+		&expect.BExp{R: "The system is going down NOW!"},
+	}, 20*time.Second)).To(Succeed())
+}
```

Comment:
Wondering if this should be elsewhere (like under `console` pkg)

Reply:
There's only one other use of this, highlighted by your next comment, and since it has a vastly different timeout I left it alone...
But I can move this to the console package if you think it adds value, I'm sure that other test doesn't really need 4 minutes :)

Reply:
Let's leave as is for the moment.
Thanks